src/Membership/AdherentRegistry.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace AppBundle\Membership;
+
+use AppBundle\CitizenAction\CitizenActionManager;
+use AppBundle\CitizenInitiative\ActivitySubscriptionManager;
+use AppBundle\CitizenInitiative\CitizenInitiativeManager;
+use AppBundle\CitizenProject\CitizenProjectManager;
+use AppBundle\Committee\CommitteeManager;
+use AppBundle\Committee\Feed\CommitteeFeedManager;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\AdherentActivationToken;
+use AppBundle\Entity\Committee;
+use AppBundle\Entity\Summary;
+use AppBundle\Event\EventManager;
+use AppBundle\Event\EventRegistrationManager;
+use AppBundle\Report\ReportManager;
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\ORM\EntityManagerInterface;
+
+class AdherentRegistry
+{
+    private $em;
+    private $manager;
+    private $citizenInitiativeManager;
+    private $citizenActionManager;
+    private $eventManager;
+    private $committeeManager;
+    private $committeeFeedManager;
+    private $activitySubscriptionManager;
+    private $citizenProjectManager;
+    private $reportManager;
+    private $registrationManager;
+
+    public function __construct(
+        EntityManagerInterface $em,
+        ObjectManager $manager,
```

Comment:
Why is this needed? You already have an `EntityManagerInterface` that should be enough.